testing/tests/DevExpress.ui.widgets/treeViewParts/expandedItems.js

```diff
@@ -264,6 +339,20 @@ module('Expanded items', {
         $toggleExpandIcon.trigger('dxclick');
     });
 
+    test('itemExpanded should be fired when expanding item with custom expander icon', function(assert) {
+        const data = $.extend(true, [], DATA[5]);
+
+        const $treeView = initTree({
+            items: data,
+            expandButtonIcon: 'add',
+            collapseButtonIcon: 'add',
+        });
+        const treeView = $treeView.dxTreeView('instance');
+        const $toggleExpandIcon = $($treeView.find('.' + internals.CUSTOM_EXPAND_ICON_CLASS).eq(0));
+        treeView.on('itemExpanded', () => assert.ok(true, 'itemExpanded was fired'));
```

Comment:
1. We can use `sinon.spy` here to make this test clearer.
2. Let's separate [arrange, action, assert](https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/) parts with an empty row.

Reply:
roger